src/data/teamMembers/backendTeam.js

```diff
@@ -1,27 +1,32 @@
 export const teamBackend = [
     {
-        name: "Aslam Dhikrullahi",
-        title: "Design Lead",
-        image: "/test_img1.jpg",
+        name: "Oluwafikayomi Soetan",
+        title: "Backend Development Lead",
+        image: "/Oluwafikayomi.png",
     },
```

Comment:
The name of the images should be in this format:
toyin_BE
precious_BE
abraham_FE
miracle_PD

etc

Reply:
okay sir